x-pack/plugins/apm/server/lib/services/get_services/get_services_items.ts

```diff
@@ -3,88 +3,80 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-
-import { mergeProjection } from '../../../../common/projections/util/merge_projection';
-import {
-  PROCESSOR_EVENT,
-  AGENT_NAME,
-  SERVICE_ENVIRONMENT,
-  TRANSACTION_DURATION,
-} from '../../../../common/elasticsearch_fieldnames';
+import { uniq } from 'lodash';
+import { arrayUnionToCallable } from '../../../../common/utils/array_union_to_callable';
 import { PromiseReturnType } from '../../../../typings/common';
 import {
   Setup,
   SetupTimeRange,
   SetupUIFilters,
 } from '../../helpers/setup_request';
 import { getServicesProjection } from '../../../../common/projections/services';
+import {
+  getTransactionDurationAvg,
+  getAgentName,
+  getTransactionRate,
+  getErrorRate,
+  getEnvironments,
+} from './get_metrics';
```

Comment:
Unless this is related to the metric based ui or agent runtime metrics we should find another term.

Reply:
Changed this to `get_services_items_stats`. Also pluralised the function names.